scripts/schema-manager/manifest-loader.ts

```diff
@@ -0,0 +1,105 @@
+import { TigrisSchema } from '@tigrisdata/core/dist/types';
+import path from 'path';
+import fs from 'fs';
+import * as Log from './logger';
+
+// TODO: allow user to override this
+const SCHEMAS_ROOT = 'lib/models';
```

Comment:
Other options - `/lib/models`, `/tigris/models`, `/lib/data`, `/db`. Looking for opinions here.